plugins/README.md

```diff
@@ -5,27 +5,35 @@ The plugins API contains the following methods:
 
 ### `wp.plugins.registerPlugin( name: string, settings: Object )`
 
+This method registers a new plugin.
+
 This method takes two arguments:
 
 1. `name`: A string identifying the plugin. Must be unique across all registered plugins.
 2. `settings`: An object containing the following data:
-   - `render`: A component containing the UI elements to be rendered. See the list below for all available UI elements.
+   - `render`: A component containing the UI elements to be rendered.
+
+See [the edit-post module documentation](../edit-post/) for available components.
 
 **Example**
 
 ```jsx
 const { Fragment } = wp.element;
-const { PluginSidebar } = wp.editPost.__experimental;
+const { PluginSidebar, PluginMoreMenuItem } = wp.editPost.__experimental;
 const { registerPlugin } = wp.plugins;
 
 const Component = () => (
 	<Fragment>
-		<PluginSidebar name="first-sidebar-name" title="My Sidebar">
+		<PluginSidebar name="sidebar-name" title="My Sidebar">
 			Content of the first sidebar
 		</PluginSidebar>
-		<PluginSidebar name="second-sidebar-name" title="My Second Sidebar">
-			Content of the second sidebar
-		</PluginSidebar>
+		<PluginMoreMenuItem
+			name="menu-item-name"
+			title="My Second Sidebar"
```

Comment:
Since there's no mention any longer of the second sidebar, `My Second Sidebar` could be changed to `My Sidebar`.